examples/panantarctic_regional_simulation.jl

```diff
@@ -0,0 +1,233 @@
+using ClimaOcean
+using ClimaOcean.ECCO
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.Grids: φnode
+using CairoMakie
+using CFTime
+using Dates
+using Printf
+using CUDA
+
+arch = GPU()
+Nx = 1440
+Ny = 400
+Nz = 40
+
+depth = 6000meters
+z = ExponentialCoordinate(Nz, -depth, 0)
+
+grid = LatitudeLongitudeGrid(arch;
+                             size = (Nx, Ny, Nz),
+                             halo = (7, 7, 7),
+                             z = z,
+                             latitude  = (-80, -20),
+                             longitude = (0, 360))
+
+bottom_height = regrid_bathymetry(grid;
+                                  minimum_depth = 10meters,
+                                  interpolation_passes = 7,
+                                  major_basins = 1)
+ 
+grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bottom_height); active_cells_map=true)
+
+h = grid.immersed_boundary.bottom_height
+
+fig, ax, hm = heatmap(h, colormap=:deep, colorrange=(-depth, 0))
+Colorbar(fig[0, 1], hm, label="Bottom height (m)", vertical=false)
+save("acc_bathymetry.png", fig)
+
+start_date = DateTime(1993, 1, 1)
+end_date   = DateTime(1993, 12, 1) 
+
+#
+# Restoring force 
+#               φS                   φN             -20
+# -------------- | ------------------ | ------------ |
+# no restoring   0    linear mask     1   mask = 1   1
+#
```

Comment:
How would this appear when literated? We can wait for the preview and see.

Reply:
https://clima.github.io/ClimaOceanDocumentation/previews/PR142/literated/panantarctic_regional_simulation/